Service/FindGithubRepositoryThroughOrganizationService.php

```diff
@@ -319,14 +277,15 @@ public function getOrganisationRepo(string $url, ObjectEntity $organization, str
      */
     private function getOrganizationCatalogi(ObjectEntity $organization): void
     {
-        if ($githubRepo = $this->getGithubRepoFromOrganization($organization->getValue('name'))) {
-            isset($this->io) && $this->io->success('Github repo found and fetched for '.$organization->getName());
-            if ($openCatalogi = $this->getOpenCatalogiFromGithubRepo($organization->getValue('name'))) {
-                isset($this->io) && $this->io->success('OpenCatalogi.yml or OpenCatalogi.yaml found and fetched for '.$organization->getName());
+        if ($this->getGithubRepoFromOrganization($organization->getValue('name')) !== null) {
+            $this->pluginLogger->debug('Github repo found and fetched for '.$organization->getName());
+            if (($openCatalogi = $this->getOpenCatalogiFromGithubRepo($organization->getValue('name'))) !== null) {
+                $this->pluginLogger->debug('OpenCatalogi.yml or OpenCatalogi.yaml found and fetched for '.$organization->getName());
 
-                if (!$openCatalogi) {
+                if ($openCatalogi === null) {
```

Comment:
Again if !$openCatalogi is not the exact same as !== null. These comments are here so we think twice before just changing these, isset or empty might be better/safer